wavenet_ops.py

```diff
@@ -32,6 +34,29 @@ def causal_conv(value, filter_, dilation, name='causal_conv'):
             restored = tf.nn.conv1d(padded, filter_, stride=1, padding='SAME')
         # Remove excess elements at the end
         result = tf.slice(restored,
-                         [0, 0, 0],
-                         [-1, tf.shape(value)[1], -1])
+                          [0, 0, 0],
+                          [-1, tf.shape(value)[1], -1])
         return result
+
+
```

Comment:
Yep, I think it's 1 line between methods or inner functions, but 2 lines for top-level functions and classes.